mirdata/core.py

```diff
@@ -216,6 +231,19 @@ def load_tracks(self):
         """
         return {track_id: self.track(track_id) for track_id in self.track_ids}
 
+    def load_multitracks(self):
+        """Load all multitracks in the dataset
+
+        Returns:
+            dict:
+                {`mtrack_id`: multitrack data}
+
+        Raises:
+            NotImplementedError: If the dataset does not support Multitracks
+
+        """
+        return {mtrack_id: self.multitrack(mtrack_id) for mtrack_id in self.mtrack_ids}
+
     def choice_track(self):
```

Comment:
do we want to add a choice_multitrack? or is it overkill?

Reply:
it could be done.